src/com/twitter/intellij/pants/execution/PantsJUnitTestRunConfigurationProducer.java

```diff
@@ -104,6 +105,11 @@ else if (testPackage != null) {
     }
 
     taskSettings.setScriptParameters(StringUtil.join(scriptParameters, " "));
+    Optional<String[]> vmOptions =
+      PantsOptions.getPantsOptions(module.getProject()).flatMap(options -> options.getList("jvm.test.junit.options"));
+
```

Comment:
Would be good to extract the string to a constant next to https://github.com/pantsbuild/intellij-pants-plugin/blob/master/common/com/twitter/intellij/pants/util/PantsConstants.java#L40?